### ISSUE TITLE
Upgrade to systemd-256 cooker isos fail to boot

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2979]
**OpenMandriva version**:
Cooker
**Describe the bug**:
Cooker isos fail to boot if made with systemd-256.1
**Steps to reproduce**:
Try Cooker isos 3138 or 3140
**Observed behavior**:
Isos fail to boot resulting in error: Failed to start initrd-switch-root.service
**Expected behavior**:
Isos should boot to graphical 'Live' environment 
**Additional comment**:
If you boot those isos in cli mode and run `ls /usr/bin` you will see that `/usr/bin` does not  have the files it should after booting an OMLx iso. This issue happens with systemd-256. If we make Cooker isos with systemd-255.7 the isos boot normally.Note: I am not saying systemd-256 is the cause, rather that this occurred and using 255.7 results in effective workaround. In other words reporting what I see with my user level knowledge.
**Logs and screenshots if relevant**
![Screenshot_20240615_131719](https://github.com/OpenMandrivaAssociation/distribution/assets/5704386/493a373c-ac06-46d0-96c4-62d1f38b44e7)

![Screenshot_20240616_094257](https://github.com/OpenMandrivaAssociation/distribution/assets/5704386/8059ab36-b2b9-4ec5-881b-258f94b10e20)

This has been discussed on OpenMandriva Chat and verified by rugyada and angry-penguin.

**Comment by benbullard79** [comment]
Cooker downgraded to systemd-255.7 and isos work again.

**Comment by benbullard79** [comment]
Today dsynced 4 Cooker systems. VM's Cooker P5, P6, and LXQt plus hw laptop Cooker P5. This problem seems fixed, `systemd-256.4-1` packages built July 25 seem working just fine here.

hw system:

```
Machine:
  Type: Laptop System: ASUSTeK product: Vivobook_ASUSLaptop M1605YA_M1605YA

```
```
CPU:
  Info: 6-core model: AMD Ryzen 5 7530U with Radeon Graphics bits: 64
```
